tests/e2e/scripts/cloud-run/deploy.sh

```diff
@@ -41,8 +41,8 @@ LOG_DIR="$(mktemp -d /tmp/log.XXXX)"
 
 # Determine names of all resources
 UNIQUE_ID=$(get_unique_id | cut -c 1-6)
-BACKEND_SERVICE_NAME=$(get_cloud_run_service_name_with_sha "${BACKEND}")
-BACKEND_SERVICE_NAME="${BACKEND_SERVICE_NAME}-${UNIQUE_ID}"
+BACKEND_SERVICE_NAME="e2e-test-${BACKEND_PLATFORM}-${BACKEND}-${UNIQUE_ID}"
+
 PROXY_SERVICE_NAME=$(get_cloud_run_service_name_with_sha "api-proxy")
```

Comment:
Suggest you change this to the same format is `BACKEND_SERVICE_NAME`.

Otherwise, ESPv2 is deployed as `cloudesf-testing-e2e-test-...` on Cloud Run. Then the janitor won't pick these up. You can remove the `cloudesf-testing-` prefix I guess.

Reply:
I removed `cloudesf-testing-` in tests/e2e/scripts/cloud-run/utilities.sh. Is that what you are saying?

Reply:
Yup, missed that. Thanks